.gitmodules

```diff
@@ -267,7 +267,10 @@
 	url = https://github.com/ClickHouse/nats.c
 [submodule "contrib/vectorscan"]
 	path = contrib/vectorscan
-	url = https://github.com/VectorCamp/vectorscan
+	# FIXME: update once upstream fixes will be merged:
+	# - https://github.com/VectorCamp/vectorscan/pull/148
+	# - https://github.com/VectorCamp/vectorscan/pull/149
+	url = https://github.com/azat-ch/vectorscan
```

Comment:
Sorry for being a bit late for the party, but there's no need to change the url. It is available all over the forks

https://github.com/VectorCamp/vectorscan/commit/aaca65aa210ce3ec91bd2b249c4d59e55e80a869

Reply:
I am on it ...

Reply:
I also don't mind too much. I would say the unspoken rule is "reference upstream if possible (= there are no custom patches), otherwise reference a fork in the ClickHouse organization".

Anyways, after making some silly mess, I now
- created https://github.com/ClickHouse/vectorscan/
- pushed your two fixes to [branch ClickHouse/vectorscan/5.4.9](https://github.com/ClickHouse/vectorscan/commits/ClickHouse/vectorscan/5.4.9) in this forked repo
- and referenced that fork + commit from ClickHouse ([here](https://github.com/ClickHouse/ClickHouse/pull/49848)).

Your two pushes against vectorscan upstream remain as is (fingers crossed they are merged).